NickvisionTubeConverter.GNOME/Views/MainWindow.cs

```diff
@@ -421,7 +421,7 @@ private MainWindow(Gtk.Builder builder, MainWindowController controller, Adw.App
         };
         try
         {
-            _unityLauncher = unity_launcher_entry_get_for_desktop_id($"{_controller.AppInfo.ID}.desktop");
+             _unityLauncher = unity_launcher_entry_get_for_desktop_id(string.IsNullOrEmpty(Environment.GetEnvironmentVariable("SNAP")) ? $"{_controller.AppInfo.ID}.desktop" : "tube-converter_tube-converter.desktop");
```

Comment:
Could you remove that extra space at the beginning of the line?

Reply:
Done